Cargo.toml

```diff
@@ -39,11 +39,11 @@ swc_core = { version = "0.83.12", features = [
 testing = { version = "0.34.1" }
 
 # Turbo crates
-turbopack-binding = { git = "https://github.com/vercel/turbo.git", tag = "turbopack-230928.3" }
+turbopack-binding = { git = "https://github.com/vercel/turbo.git", tag = "turbopack-230928.5" }
 # [TODO]: need to refactor embed_directory! macro usages, as well as resolving turbo_tasks::function, macros..
-turbo-tasks = { git = "https://github.com/vercel/turbo.git", tag = "turbopack-230928.3" }
+turbo-tasks = { git = "https://github.com/vercel/turbo.git", tag = "turbopack-230928.5" }
 # [TODO]: need to refactor embed_directory! macro usage in next-core
-turbo-tasks-fs = { git = "https://github.com/vercel/turbo.git", tag = "turbopack-230928.3" }
+turbo-tasks-fs = { git = "https://github.com/vercel/turbo.git", tag = "turbopack-230928.5" }
```

Comment:
The version need to be updated in more places. e. g. `packages/next/package.json` and `pnpm-lock.yaml`. Please use the script to update correctly